templates/branding/openSUSE/external_reporting.html.ep

```diff
@@ -35,6 +35,9 @@
 %         if ($subproduct eq 'Server' && $version eq '12') {
 %             $version = '12 (SLES 12)';
 %         }
+%         if ($subproduct eq 'Desktop-Updates' && $version eq '12 SP1') {
```

Comment:
I also wondered about "Desktop" but when you take a look on bugzilla.suse.com you will see that SLES and SLED are inconsistent. Probably it should be `$subproduct =~ /Desktop/`. About the `$version` I think it's correct how it is